ush/cpld_gridgen.sh

```diff
@@ -3,96 +3,104 @@ set -eux
 
 function edit_namelist {
 
-  sed -e "s/NI_GLB/$NI/g" \
-      -e "s/NJ_GLB/$NJ/g" \
-      -e "s|FIXDIR|$FIXDIR_PATH|g" \
-      -e "s|OUTDIR|$OUTDIR_PATH|g" \
-      -e "s|MOSAICDIR|$MOSAICDIR_PATH|g" \
-      -e "s/TOPOGFILE/$TOPOGFILE/g" \
-      -e "s/EDITSFILE/$EDITSFILE/g" \
-      -e "s/RESNAME/$RESNAME/g" \
-      -e "s/MOSAICRES/$MOSAICRES/g" \
-      -e "s/NPX/$NPX/g" \
-      -e "s/DO_MASKEDIT/$MASKEDIT/g" \
-      -e "s/DO_DEBUG/$DEBUG/g" \
-      -e "s/DO_POSTWGTS/$DO_POSTWGTS/g"
+    sed -e "s/NI_GLB/$NI/g" \
+	-e "s/NJ_GLB/$NJ/g" \
+	-e "s|FIXDIR|$FIXDIR_PATH|g" \
+	-e "s|OUTDIR|$OUTDIR_PATH|g" \
+	-e "s|MOSAICDIR|$MOSAICDIR_PATH|g" \
+	-e "s/TOPOGFILE/$TOPOGFILE/g" \
+	-e "s/EDITSFILE/$EDITSFILE/g" \
+	-e "s/RESNAME/$RESNAME/g" \
+	-e "s/MOSAICRES/$MOSAICRES/g" \
+	-e "s/NPX/$NPX/g" \
+	-e "s/DO_MASKEDIT/$MASKEDIT/g" \
+	-e "s/DO_DEBUG/$DEBUG/g" \
+	-e "s/DO_POSTWGTS/$DO_POSTWGTS/g"
 }
 
 export RESNAME=${RESNAME:-$1}
 export DEBUG=.false.
 export MASKEDIT=.false.
 export DO_POSTWGTS=.false.
-export OUTDIR_PATH=${OUTDIR_PATH:-/scratch2/NCEPDEV/climate/Denise.Worthen/grids-20220116}
+export OUTDIR_PATH=${OUTDIR_PATH:-/scratch1/NCEPDEV/climate/Denise.Worthen/grids-20220116}
 export MOSAICDIR_PATH=${MOSAICDIR_PATH:-$PATHTR/fix/orog}
-APRUN=${APRUN:-"srun"}
-
-if [ $RESNAME = 400 ]; then
-  echo "The 4 degree resolution is not implemented yet"
-  exit 1
+if [[ $MOSAICRES == C3072 ]]; then
+    export NPX=3072
+elif [[ $MOSAICRES == C1152 ]]; then
+    export NPX=1152
+elif [[ $MOSAICRES == C768 ]]; then
+    export NPX=768
+elif [[ $MOSAICRES == C384 ]]; then
+    export NPX=384
+elif [[ $MOSAICRES == C192 ]]; then
+  export NPX=192
+elif [[ $MOSAICRES == C096 ]]; then
+  export MOSAICRES=C96
+  export NPX=96
+elif [[ $MOSAICRES == C048 ]]; then
+  export MOSAICRES=C48
+  export NPX=48
+fi
+if [ $RESNAME = 500 ]; then
+  export FIXDIR_PATH=/scratch1/NCEPDEV/global/glopara/fix/mom6/20220805/500/
```

Comment:
This looks like a Hera directory. How does this work on another machine?

Reply:
I wasn't sure if you wanted to add the required 5deg ocean files to the usual location since we're not running baselines on for the 5deg case. There is no use-case really for the 5-deg ocean other than the GW infrastructure work Rahul is doing.

Reply:
Ok. Does this code have a large user base or is it just you right now? If the former, do you want to add a comment about running the 5-deg on Hera only?

Reply:
One of the reasons I'm making this PR is so that other users can generate their own configurations. Right now, they simply ask me to generate what they need for them. Once we get this committed, I'll try steering users to ufs-utils instead. 

I will add a note that 5-deg is only on hera.